src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/MySqlEngine.java

```diff
@@ -764,6 +768,63 @@ protected boolean checkConnection(final Connection conn) {
         }
     }
 
+    @Override
+    protected String getSchema() throws DatabaseEngineException {
+        try {
+            return this.conn.getCatalog();
```

Comment:
Why **getCatalog** instead of **getSchema** (the default method call on the superclasse)?

Reply:
For MySql schema and catalog are the same thing.
However, for some reason _getSchema()_ doesn't work, just returns _null_ :(